.ado/pipelines/templates/stages-loadtest-azure.yaml

```diff
@@ -212,9 +214,15 @@ stages:
                 Invoke-WebRequest $logsUrl -OutFile "results/logs.zip"
 
                 if ($result.testResult -eq "FAILED") {
-                  throw "*** ERROR: Test $($result.testRunId) failed."
+                  throw "*** ERROR: Test result for test run $($result.testRunId) is failed. Test did not match the defined test criteria."
                 }
 
+              } elseif ($result.status -eq "FAILED") {
+                # test ended in failed state
+                throw "*** ERROR: Test run $(testRunId) ended in failed state."
+              } else {
+                # test is still running
```

Comment:
what about Cancelled (manually)?

Reply:
The API documentation does not contain the list of potential states (last time i checked). Not sure if there is a cancelled state have to manually test that.

Reply:
Worst thing that can happen though is that this job runs into a timeout (after 60 mins).

Reply:
Added 'CANCELLED' state.

Reply:
great!